enterprise/static/enterprise/enterprise_course_enrollment_page.js

```diff
@@ -0,0 +1,60 @@
+function focusableElementsSelector() {
+    return [
+        'a[href], area[href], input:not([disabled]), select:not([disabled]),',
+        'textarea:not([disabled]), button:not([disabled]), iframe, object, embed,',
+        '*[tabindex], *[contenteditable]'
+    ].join(' ');
+}
+function enableTabIndexingOn(containerSelector) {
+    var container = $(containerSelector),
+        focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible');
+    container.attr('aria-hidden', 'false');
+    focusableItems.attr('tabindex', '0');
+}
+function disableTabIndexingOn(containerSelector) {
+    var container = $(containerSelector),
+    focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible');
+    container.attr('aria-hidden', 'true');
+    focusableItems.attr('tabindex', '-1');
+}
```

Comment:
@itsjeyd I named it `setTabIndexing` and sent `ariaHidden` and `tabIndex` as parameters to avoid the conditional.